Source/m_menu.c

```diff
@@ -3220,27 +3221,11 @@ setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
 
 static void M_UpdateCrosshairItems (void)
 {
-    if (hud_crosshair)
-    {
-        stat_settings2[8].m_flags  &= ~S_DISABLE;
-        stat_settings2[9].m_flags &= ~S_DISABLE;
-        stat_settings2[10].m_flags &= ~S_DISABLE;
-        if (hud_crosshair_target)
-        {
-            stat_settings2[11].m_flags &= ~S_DISABLE;
-        }
-        else
-        {
-            stat_settings2[11].m_flags |= S_DISABLE;
-        }
-    }
-    else
-    {
-        stat_settings2[8].m_flags  |= S_DISABLE;
-        stat_settings2[9].m_flags |= S_DISABLE;
-        stat_settings2[10].m_flags |= S_DISABLE;
-        stat_settings2[11].m_flags |= S_DISABLE;
-    }
+    DISABLE_ITEM(!hud_crosshair, stat_settings2[8]);
```

Comment:
So, crosshair per-se is allowed in strict mode?

Reply:
Yes, crosshairs are allowed (you can physically draw a dot in the center of the screen).